### PR TITLE
Query multiple tables.

### PR DESCRIPTION
**Pull request by Alejandroq12**
Hello, dear reviewer.
Here are all the requirements listed:

- Write queries (using `JOIN`) to answer the following questions:

- What animals belong to Melody Pond?
Screenshot from 2023-11-25 14-02-20

--------------------------------

 - List of all animals that are pokemon (their type is Pokemon).
Screenshot from 2023-11-25 14-02-59

--------------------------------

 - List all owners and their animals, remember to include those that don't own any animal.
Screenshot from 2023-11-25 14-04-01

--------------------------------

 - How many animals are there per species?
Screenshot from 2023-11-25 14-04-35

--------------------------------

- List all Digimon owned by Jennifer Orwell.
Screenshot from 2023-11-25 14-05-37

--------------------------------

- List all animals owned by Dean Winchester that haven't tried to escape.
Screenshot from 2023-11-25 14-06-19

--------------------------------

- Who owns the most animals?
Screenshot from 2023-11-25 14-07-09

--------------------------------

Thank you very much for your valuable time and feedback.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -45,3 +45,11 @@ SELECT AVG(weight_kg) FROM animals;
...
 SELECT species, AVG(escape_attempts) AS avg_escape_attempts FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+SELECT animals.name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name = 'Melody Pond';
+SELECT animals.name FROM animals JOIN species ON animals.species_id = species.id WHERE species.name = 'Pokemon';
+SELECT owners.full_name, animals.name FROM owners LEFT JOIN animals ON owners.id = animals.owner_id;
+SELECT species.name, COUNT(animals.id) FROM species LEFT JOIN animals ON species.id = animals.species_id GROUP BY species.name;
+SELECT animals.name FROM animals JOIN owners ON animals.owner_id = owners.id JOIN species ON animals.species_id = species.id WHERE owners.full_name = 'Jennifer Orwell' AND species.name = 'Digimon';
+SELECT animals.name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
+SELECT owners.full_name, subquery.animal_count FROM owners JOIN (SELECT owner_id, COUNT(*) as animal_count FROM animals GROUP BY owner_id ORDER BY animal_count DESC LIMIT 1) subquery ON owners.id = subquery.owner_id;
```
Comment (Qoosim) [inline queries.sql:55]: - [ ] Great job so far. However, the query to get `Who owns the most animals?` seems to not correct. Your output is different from the expected result. Kindly check the screenshot below to see the expected output for that query :point_down: 

### Project requirement.
proj-rqt

### Output
result

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @Alejandroq12, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  Implemented some project requirements. :heavy_check_mark: 
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Qoosim) [inline queries.sql:55]: - [ ] Great job so far. However, the query to get `Who owns the most animals?` seems to not correct. Your output is different from the expected result. Kindly check the screenshot below to see the expected output for that query :point_down: 

### Project requirement.
proj-rqt

### Output
result

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @Alejandroq12, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  Implemented some project requirements. :heavy_check_mark: 
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Alejandroq12) [reply queries.sql:55]: @Qoosim Hello, I hope you are having a good day.

Please, Could you kindly check becuse the questions says "Who owns the most animals?" And Melody Pondy owns 3 animals which is a bigger number of animals and of instance it answers the question correctly.

**Comment by Alejandroq12** [comment]
Comment for the next reviewer. 
Hello, dear reviewer.
Please take into account that the previous reviewer said that the questions "Who owsn ths mos animals?" is not correctly answer but that questions is correctly answered. Because it correctly shows the person who owns more animals wich is the correct answer

Thank you. Have a nice day.

**Review by DeliceLydia — APPROVED** [review]
## Hi @Alejandroq12, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alejandroq12** [comment]
Thank you very much for your valuable time.

**Pull request merged**